tensorflow/tools/ci_build/ci_build.sh

```diff
@@ -80,7 +80,7 @@ fi
 
 # cmake (CPU) builds do not require configuration.
 if [[ "${CONTAINER_TYPE}" == "cmake" ]]; then
-  CI_COMMAND_PREFIX=""
+  CI_COMMAND_PREFIX=("")
```

Comment:
This kind of unintended behavior is what this PR tries to fix.
There are so many things seriously wrong with Bash.
It is just way too easy to make mistakes.